pages/treeherder.py

```diff
@@ -152,6 +207,22 @@ def open_perfherder_page(self):
     def open_repos_menu(self):
         self.find_element(*self._repos_menu_locator).click()
 
+    def pin_job_and_enter_bug_number(self, bug_number):
```

Comment:
The pin_job_and_enter_bug_number test was removed as it required a log in. Although the shortcut opens the pinboard, I don't think this should be tested as the pinned job and bug cannot be saved.